docs/API.md

```diff
@@ -312,5 +412,88 @@ Or `.static()` ...
 Or even `.compose()` ...
 
 ```js
-  var obj = abstractStamp.compose(concreteStamp, additionalStamp, utilityStamp).create();
+  const obj = abstractStamp.compose(concreteStamp, additionalStamp, utilityStamp).create();
+```
+
+
+## Chaining methods
```

Comment:
However, some examples are ES5, some are ES6. No good. :-\

Okay. I'll spend some more time on that and ES6-ify all the text.